roles/addons/diskless/files/livenet_module.py

```diff
@@ -609,11 +609,14 @@ def generate_ipxe_boot_file(self):
         """Generate an ipxe boot file for the image."""
         logging.info('Creating image \'' + self.name + '\' IPXE boot file')
 
+        # selinux should be 0 (False) or 1 (True)
+        image_selinux = '1' if self.selinux else '0'
+
         # Format image ipxe boot file template with image attributes
         file_content = self.__class__.get_boot_file_template().format(image_name=self.name,
                                                                       image_initramfs=self.image,
                                                                       image_kernel=self.kernel,
-                                                                      image_selinux=self.selinux)
+                                                                      image_selinux=image_selinux)
```

Comment:
I would use `int(self.selinux)` and remove the lines 612-614 above.

Reply:
I have tested `int(self.selinux)` and it works, will add a commit soon.

Reply:
It worked this way in 1.3. 😄